dist/build_deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+IMAGE_BUILD="${IMAGE_BUILD:-ekidd/rust-musl-builder:1.30.1}"
+IMAGE="quay.io/app-sre/cincinnati"
```

Comment:
I meant to ask you to make this overridable too, or doesn't it make sense?

Reply:
I don't think this makes a lot of sense since image is public...